### PR TITLE
[REG-1672] Document new playback toolbar

### PR DESCRIPTION
**Pull request by RG-nAmKcAz**
- Documents new playback toolbar
- Adds documentation mentions for new performance stats
- Updates folder naming convention

---

Find the pull request instructions [here](https://www.notion.so/regressiongg/Pull-Request-Process-0d57a7eb582a446983edfacafa406f1e?pvs=4)

Every reviewer and the owner of the PR should consider these points in their request (feel free to copy this checklist so you can fill it out yourself in the overall PR comment)

- [ ] The code is extensible and backward compatible
- [ ] New public interfaces are extensible and open to backward compatibility in the future
- [ ] If preparing to remove a field in the future (i.e. this PR removes an argument), the argument stays but is no longer functional, and attaches a deprecation warning. A linear task is also created to track this deletion task.
- [ ] Non-critical or potentially modifiable arguments are optional
- [ ] Breaking changes and the approach to handling them have been verified with the team (in the Linear task, design doc, or PR itself)
- [ ] The code is easy to read
- [ ] Unit tests are added for expected and edge cases
- [ ] Integration tests are added for expected and edge cases
- [ ] Functions and classes are documented
- [ ] Migrations for both up and down operations are completed
- [ ] A documentation PR is created and being reviewed for anything in this PR that requires knowledge to use
- [ ] Implications on other dependent code (i.e. sample games and sample bots) is considered, mentioned, and properly handled
- [ ] Style changes and other non-blocking changes are marked as non-blocking from reviewers

### CODE REVIEW

File: docs/gameplay-sessions/gameplay-sessions-usage.mdx [hunk 1]
```diff
@@ -16,24 +16,48 @@ Sessions.
...
 This will launch a file explorer at the location of the recordings folder on your device.
 Navigate into the folder of the recording you'd like to replay, select its `data.zip` file, and then click "Load Replay".
-Once you've selected a replay, click the Play button below the Regression Games logo to begin playback.
 
 ![Open the Gameplay Session Selector](img/overlay_click_load.png)
 ![Select the Gameplay Session](img/overlay_choose_session.png)
 
+### Playing a Recording
+Once you've selected a replay, click either the Loop or Play button below on the Regression Games toolbar to begin playback.
+- The Loop function will continue to play the recording over and over until a failure occurs.
```
Comment (abeizer) [inline docs/gameplay-sessions/gameplay-sessions-usage.mdx:39]: We may want to mention that the recording needs to be able to get back to an initial state on its own, ie our SDK is not responsible for "resetting" the game to a valid state between loops

**Review by abeizer — APPROVED** [review]
